tests/snmpsim/allied.snmprec

```diff
@@ -1,2 +0,0 @@
-1.3.6.1.2.1.1.1.0|4|AT-GS950/24
```

Comment:
You should leave this check in place and just rename it to `tests/snmpsim/alliedtelesis_gs950.snmprec`